packages/yoastseo/src/scoring/assessments/inclusiveLanguage/InclusiveLanguageAssessment.js

```diff
@@ -63,7 +63,11 @@ export default class InclusiveLanguageAssessment extends Assessment {
 	 */
 	isApplicable( paper, researcher ) {
 		const memoizedTokenizer = researcher.getHelper( "memoizedTokenizer" );
-		const sentences = getSentences( paper.getText(), memoizedTokenizer );
+		const text = paper.getText();
+		const textTitle = paper.getTextTitle();
+		const sentences = getSentences( text, memoizedTokenizer );
+		// Also include the text title in the analysis as a separate sentence.
+		sentences.push( textTitle );
```

Comment:
```suggestion
		const sentences = getSentences( text, memoizedTokenizer );
		
		// Also include the text title in the analysis as a separate sentence.
		const textTitle = paper.getTextTitle();
		sentences.push( textTitle );
```

To me it makes more sense to format the code like this. But that is open for interpretation.